osprey/processes/wps_convolution.py

```diff
@@ -0,0 +1,129 @@
+from pywps import Process, LiteralInput, ComplexOutput, FORMATS
+from pywps.app.Common import Metadata
+
+from rvic.convolution import convolution
+from rvic.core.config import read_config
+from rvic.version import version
+
+from datetime import datetime, timedelta
+
+import os
+import logging
+import json
+
+pywps_logger = logging.getLogger("PYWPS")
+stderr_logger = logging.getLogger(__name__)
+
+
+def log_handler(process, response, message, process_step=None, level="INFO"):
+    if process_step:
+        status_percentage = process.status_percentage_steps[process_step]
+    else:
+        status_percentage = response.status_percentage
+
+    # Log to all sources
+    pywps_logger.log(getattr(logging, level), message)
+    stderr_logger.log(getattr(logging, level), message)
```

Comment:
It's obvious to us, but maybe make a note in a comment here that this will turn into an import.